src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const periodDiscounts = { 3: 20, 7: 50 };
+
+  const discount = getDiscount(days, periodDiscounts);
+
+  return days * pricePerDay - discount;
+}
+
+/**
+ * @param {number} days
+ * @param {Object} periodDiscounts
+ *
+ * @return {number}
+ */
+function getDiscount(days, periodDiscounts) {
+  const allPeriods = Object.getOwnPropertyNames(periodDiscounts);
```

Comment:
Using `Object.getOwnPropertyNames` is not necessary here. You can directly use `Object.keys` to get the keys of the `periodDiscounts` object as strings.